.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "root": true,
+
+  "extends": "jquery",
+
+  "reportUnusedDisableDirectives": true,
+
+  "parserOptions": {
+    "ecmaVersion": 2018
+  },
+
+  "env": {
+    "es6": true,
+    "node": true
+  },
+
+  "rules": {
+//    "strict": [ "error", "global" ],
```

Comment:
We might want to remove this

Reply:
@fnagel I was thinking about it but at the same time I wanted to somehow record the intended end state. For Node files usually a global `strict` version is preferred, for browser ones the `function` setting is better. If I just remove the code pieces, I'll need to remember about that as just removing the `"strict": "off"` part won't do anything; the rule is not enabled by default in the preset precisely because of this variance.

Thoughts?

Reply:
I'm not a big fan of unused code but I'm fine if we could add a comment about why this is here.

Reply:
OK, I just fixed this violation, no need for commented out rules anymore. ;)